ci/github/action-build-and-push/print_as_header.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+
+def assemble_header(message, default_length=80):
+    total_message_length = 4 + len(message)
+    computed_length = max(default_length, total_message_length)
+
+    spaces_length = default_length - len(message) - 4
+    print("-" * computed_length)
+    print("| " + message + " " * spaces_length + " |")
+    print("-" * computed_length)
+
+def main():
```

Comment:
MINOR: I also like having a separate free-function for main. This even allows testing separatly the main entrypoint... but in order to be totally coherent with this idea, I also expose the arguments of ``sys``, i.e.

```python
from typing import Optional, List, Any

def main(args: Optional[List[Any]]=None):
     if args is None:
         args = sys.argv[1:]

```

As I said, it does not change the behavior in this case but it is coherent with the idea of having a separate ``main`` for the main entrypoint

Reply:
I like this suggestions. I will implement it for future reference.

**Pleas note:** I am leveraging the Python from the GitHub action's base image which is 3.4 and do not intend updating it. No typing support will be added here.